video_xblock/backends/vimeo.py

```diff
@@ -55,7 +50,8 @@ def get(self, url, headers=None, can_retry=False):
         Returns:
             Response in python native data format.
         """
-        headers_ = {'Authorization': 'Bearer ' + str(self.access_token)}
+        headers_ = {'Authorization': 'Bearer ' + str(self.access_token),
+                    'Accept': 'application/json'}
```

Comment:
Please follow [Open edX JavaScript code style guide](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html) for breaking long lines.